scan/src/components/modal/ValidatorStakingInfo.re

```diff
@@ -0,0 +1,168 @@
+      boxShadow(Shadow.box(~x=`zero, ~y=`px(2), ~blur=`px(8), Css.rgba(0, 0, 0, 0.08))),
+    ]);
+
+  let cFlex = style([display(`flex), flexDirection(`column), alignItems(`flexEnd)]);
+  let rFlex = style([display(`flex), flexDirection(`row)]);
+  let balance = style([minWidth(`px(150)), justifyContent(`flexEnd)]);
+  let button = wid =>
+    style([
+      display(`flex),
+      backgroundColor(Colors.purple1),
+      width(`px(wid)),
+      height(`px(25)),
+      justifyContent(`center),
+      alignItems(`center),
+      borderRadius(`px(4)),
+      boxShadow(Shadow.box(~x=`zero, ~y=`px(4), ~blur=`px(4), rgba(11, 29, 142, 0.1))),
+      cursor(`pointer),
+    ]);
+};
+
+let stakingBalanceDetail = (title, amount, usdPrice) => {
+  <Row alignItems=Css.flexStart>
+    <Col size=1.2>
+      <Text value=title size=Text.Sm height={Text.Px(18)} spacing={Text.Em(0.03)} nowrap=true />
+    </Col>
+    <Col size=0.6>
+      <div className=Styles.cFlex>
+        <div className=Styles.rFlex>
+          <Text
+            value={amount |> Format.fPretty}
+            size=Text.Lg
+            weight=Text.Semibold
+            spacing={Text.Em(0.02)}
+            nowrap=true
+            code=true
+          />
+          <HSpacing size=Spacing.sm />
+          <Text
+            value="BAND"
+            size=Text.Lg
+            code=true
+            weight=Text.Thin
+            spacing={Text.Em(0.02)}
+            nowrap=true
+          />
+        </div>
+        <VSpacing size=Spacing.xs />
+        <div className={Css.merge([Styles.rFlex, Styles.balance])}>
+          <Text
+            value={amount *. usdPrice |> Format.fPretty}
+            size=Text.Sm
+            spacing={Text.Em(0.02)}
+            weight=Text.Thin
+            nowrap=true
+            code=true
+          />
+          <HSpacing size=Spacing.sm />
+          <Text
+            value="USD"
+            size=Text.Sm
+            code=true
+            spacing={Text.Em(0.02)}
+            weight=Text.Thin
+            nowrap=true
+          />
+        </div>
+      </div>
+    </Col>
+  </Row>;
+};
+
+[@react.component]
+let make = (~delegatorAddress, ~validatorAddress) =>
+  {
+    let infoSub = React.useContext(GlobalContext.context);
+    let balanceAtStakeSub =
+      DelegationSub.getTotalStakeByDelegatorValidator(delegatorAddress, validatorAddress);
+    let unbondingSub =
+      UnbondingSub.getUnbondingBalanceByValidator(delegatorAddress, validatorAddress);
+    let unbondingListSub = UnbondingSub.getUnbondingList(delegatorAddress, validatorAddress);
+
+    let%Sub info = infoSub;
+    let%Sub balanceAtStake = balanceAtStakeSub;
+    let%Sub unbonding = unbondingSub;
+    let%Sub unbondingList = unbondingListSub;
+
+    let balanceAtStakeAmount = balanceAtStake.amount->Coin.getBandAmountFromCoin;
+    let unbondingAmount = unbonding->Coin.getBandAmountFromCoin;
+    let rewardAmount = balanceAtStake.reward->Coin.getBandAmountFromCoin;
```

Comment:
Let's change to `let balanceAtStakeAmount = balanceAtStake.amount` and move `Coin.getBandAmountFromCoin` into `stakingBalanceDetail`.